.github/ISSUE_TEMPLATE/cut_release.md

```diff
@@ -34,15 +37,22 @@ To find more information on specific steps, please see documentation [here](http
 - [ ] Release Captain: Update RKE2
 - [ ] Release Captain: Tag new RKE2 RC
 - [ ] Release Captain: Tag new RKE2 packaging RC "testing"
+- [ ] Release Captain: Prepare PRs as needed to update [KDM](https://github.com/rancher/kontainer-driver-metadata/) in the appropriate dev branches using an RC.  For more information on the structure of the PR, see the [docs](https://github.com/rancher/rke2/blob/master/developer-docs/upgrading_kubernetes.md#update-rancher-kdm)
+  - [ ] If server args, agent args, or charts are changed, link relevant rancher/rancher issue or create new rancher/rancher issue
+  - [ ] If any new issues are created, escalated to Rancher PJM so they know and can plan for it 
+- [ ] EM: Review and merge above PR
```

Comment:
The only point of clarification I think would be helpful is to mention expectations after this step for QA. 

QA should point KDM to dev branch and use these rc's. Does this also mean that if a server/agent/chart arg is not presented correctly in rancher, we should hold up release until we fix KDM? What if it's net new and there's a Rancher UI change that needs to happen as well to receive it correctly?

Reply:
thanks max.

i will add the qa bit.

> Does this also mean that if a server/agent/chart arg is not presented correctly in rancher, we should hold up release until we fix KDM? What if it's net new and there's a Rancher UI change that needs to happen as well to receive it correctly?

No, it means that if there are changes, we need to at least consider them and open issues. A change that is a rename or delete of a flag is likely going to BREAK functionality. A net new add, if its not explicitly needed in the next release of rancher doesn't need to hold up that release, it just needs to get onto someone's radar, so an issue needs opened.

Reply:
@rancher-max I followed up with a line for QA. please review